backend/src/main/java/com/codesquad/airbnb/core/charge/ChargeManager.java

```diff
@@ -0,0 +1,59 @@
+package com.codesquad.airbnb.core.charge;
+
+import com.codesquad.airbnb.core.charge.discount.DiscountBill;
+import com.codesquad.airbnb.core.charge.discount.DiscountPolicy;
+import com.codesquad.airbnb.core.common.embeddable.GuestGroup;
+import com.codesquad.airbnb.core.common.embeddable.StayDate;
+import com.codesquad.airbnb.core.room.entity.Room;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class ChargeManager {
+
+    private static final double SERVICE_COMMISSION_RATE = 14.0;
+    private static final double TAX_COMMISSION_RATE = 1.4;
+
+    private final List<DiscountPolicy> discountPolicies;
+
+    public ChargeBill createBill(Room room, StayDate stayDate, GuestGroup guestGroup) {
+        Duration duration = Duration.between(
+            stayDate.getCheckinDate().atStartOfDay(),
+            stayDate.getCheckoutDate().atStartOfDay()
+        );
+
+        return new ChargeBill(
+            (int) duration.toDays(),
+            room.getPrice().getLodging(),
```

Comment:
```suggestion
            room.getLodgingPrice(),
```
이런 식으로 프로퍼티를 구성하면, room 내부에 있는 Price 타입의 필드에 영향을 받지 않을 것 같습니다
디미터 법칙을 찾아보시면 좋을 것 같아요!

Reply:
아하 감사합니다!!